src/helper/Helper.tsx

```diff
@@ -0,0 +1,23 @@
+import { CompletedFilter } from '../types/CompletedFilter';
+import { Todo } from '../types/Todo';
+
+export const filteredTodos = (
+  todos: Todo[],
+  completedFilter: CompletedFilter,
+) => {
+  return todos.filter(todo => {
```

Comment:
Let's don't filter the array if `completedFilter` equals CompletedFilter.All

Reply:
done, I made these corrections already in the third part